src/store/reducers/storage/utils.ts

```diff
@@ -232,7 +258,8 @@ export const prepareStorageNodesResponse = (data: TNodesInfo): PreparedStorageRe
         return undefined;
     }).filter((group): group is TableGroup => Boolean(group));
 
-    const preparedNodes = Nodes?.map(prepareStorageNodeData);
+    const maximumSlots = calculateMaximumSlotsPerDisk(Nodes, MaximumSlotsPerDisk);
+    const preparedNodes = Nodes?.map((node) => prepareStorageNodeData(node, maximumSlots));
```

Comment:
It looks like different table chunks could have different pdisk size with such calculations, since `prepareStorageNodesResponse` is called for every chunk

Reply:
It handles corner case of the scenario, which it a corner case by itself. Decided in pm to leave it as it is in PR